shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -165,7 +170,7 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result;
 #pragma mark -
 
 @implementation FlutterEngineTerminationHandler {
-  FlutterEngine* _engine;
+  __weak FlutterEngine* _engine;
```

Comment:
This fixes a retain cycle that will affect essentially all real-word, non-add-to-app cases (and was breaking my new test).

Reply:
Thanks for spotting this.